lib/nano/cw_nano.dart

```diff
@@ -309,92 +203,26 @@ class CWNanoUtil extends NanoUtil {
   BigInt rawPerBanano = BigInt.parse("100000000000000000000000000000");
   BigInt rawPerXMR = BigInt.parse("1000000000000");
   BigInt convertXMRtoNano = BigInt.parse("1000000000000000000");
-  // static BigInt convertXMRtoNano = BigInt.parse("1000000000000000000000000000");
-
-  /// Convert raw to ban and return as BigDecimal
-  ///
-  /// @param raw 100000000000000000000000000000
-  /// @return Decimal value 1.000000000000000000000000000000
-  ///
-  Decimal _getRawAsDecimal(String? raw, BigInt? rawPerCur) {
-    rawPerCur ??= rawPerNano;
-    final Decimal amount = Decimal.parse(raw.toString());
-    final Decimal result = (amount / Decimal.parse(rawPerCur.toString())).toDecimal();
-    return result;
-  }
 
   @override
   String getRawAsDecimalString(String? raw, BigInt? rawPerCur) {
-    final Decimal result = _getRawAsDecimal(raw, rawPerCur);
+    final Decimal result = NanoAmounts.getRawAsDecimal(raw, rawPerCur);
     return result.toString();
```

Comment:
this function is only ever used when converting to fiat price, so there's no risk of the large amount bug (that was a separate issue), but looking at the 1 place where this function is used we can actually remove this function entirely by instead calling the imprecise version of the function since we don't need the precision anyways (since the function that converts to fiat uses a double anyways)